src/hono-base.ts

```diff
@@ -69,6 +69,23 @@ export type HonoOptions<E extends Env> = {
    * @see https://hono.dev/api/hono#router-option
    */
   router?: Router<[H, RouterRoute]>
+  /**
+   * `getPath` option can handle the host header value.
```

Comment:
adjust to the format
```suggestion
   * `getPath` can handle the host header value.
   * @example
```